.github/workflows/build.yml

```diff
@@ -62,21 +65,53 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-          # TODO(hawflau): investigate and fix flakiness on Windows
-          # - windows-latest
+          - windows-latest
         python:
           - "3.7"
           - "3.8"
           - "3.9"
-        rust:
-          - stable
+        # folders that is commented below requires credentials, no need to spare time to run them
+        tests_folder:
```

Comment:
(non-blocking) question - do we have any plan to load balance test load under each test folder or across all test cases? My worry is some folders contains lot more test cases than others. So overall running all tests under that particular test folder will take much more time than others combined

Reply:
That is a valid point. I think we can give a chance to `pytest-split` (https://jerry-git.github.io/pytest-split/) which creates test groups from their time to completion.